extras/help_generator/writers.py

```diff
@@ -59,10 +59,10 @@ def write_help_html(doc_dic, helpdir, fname, sli_command_list, keywords):
     hlplist = []
     name = ''
 
-    for key, value in doc_dic.iteritems():
+    for key, value in doc_dic.items():
```

Comment:
Ok, I thought they would find a clever way to do this in py2... but if the list is created, then there is indeed really no point